website/docs/tutorials/03-project-initialization/02-protostar-toml-v2.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_label: protostar.toml V2
+---
+# `protostar.toml` V2
+
+```toml title="protostar.toml example"
+[project]
+protostar-version = "PROTOSTAR_VERSION"     # required
+
+# Shared Configuration (affects all commands)
+lib-path = "lib"                            # install, update, remove
+cairo-path = ["lib/cairo-contracts/src"]    # build, test
+
+# Defines contract names. Currently, required by the build command.
+[contracts]
+main = ["src/feature_a.cairo", "src/feature_b.cairo"]   
+proxy = ["src/proxy.cairo"]
+account = ["src/account.cairo"]
+
+# Command Configuration
+[format]
+target = ["src", "tests"]
+
+[declare]
+network = "testnet"
+
+# Command Configuration Profile
+[profile.mainnet.declare]
+network = "mainnet"
+
+# Shared Configuration Profile
+[profile.devnet.project]
+gateway-url = "http://127.0.0.1:5050/"
+chain-id = 1536727068981429685321
+```
+
+
+### `[project]`
+#### `protostar-version`
+This attribute is used to check the compatibility between Protostar and the project, which is useful when many developers work on the same project or going back to Protostar project after a long break. 
```

Comment:
Protostar doesn't check the compatibility itself. It can be used to assess this.

Reply:
Updated the description